py-polars/polars/exceptions.py

```diff
@@ -72,16 +72,52 @@ class SchemaError(PolarsError):  # type: ignore[no-redef, misc]
         """Exception raised when an unexpected schema mismatch causes an error."""
 
     class SchemaFieldNotFoundError(PolarsError):  # type: ignore[no-redef, misc]
-        """Exception raised when a specified schema field is not found."""
+        """
+        Exception raised when a specified schema field is not found.
+
+        Example
+        -------
```

Comment:
The sections must be "Examples" rather than "Example". This is important, because otherwise the examples won't run. Could you update all the docstrings in this module accordingly?

```suggestion
        Examples
        --------
```